buildSrc/src/main/groovy/com/uber/okbuck/rule/base/GenRule.groovy

```diff
@@ -18,31 +14,21 @@ final class GenRule extends BuckRule {
             String output = "${name}_out",
             boolean executable = false) {
         super(RuleType.GENRULE, name)
-        this.inputs = inputs
-        this.bashCmds = bashCmds
-        this.globSrcs = globSrcs
-        this.output = output
-        this.executable = executable
+        genrule = new template.base.GenRule()
```

Comment:
Is it normal for the class to be a nonconventional lowercase `template`?

Reply:
This is just the FQCN as there was a class name conflict

Reply:
The current class will go away and we can drop the FQCN in future PRs